crates/rattler_installs_packages/src/wheel_builder/build_environment.rs

```diff
@@ -158,6 +160,8 @@ impl<'db> BuildEnvironment<'db> {
 
         Command::new(self.venv.python_executable())
             .current_dir(&self.package_dir)
+            // pass all env variables defined by user
+            .envs(&self.envs_variables)
```

Comment:
What happens if we have `PATH` itself as environment variable?

Reply:
I've changed the logic so PATH passed in env_variables map have priority over the one from environment. Let me know if it's ok